spirit/category/admin/forms.py

```diff
@@ -36,3 +37,10 @@ def clean_parent(self):
                                               "can not have a parent since it has childrens"))
 
         return parent
+
+    def clean_color(self):
+        color = self.cleaned_data["color"]
+
+        if color and not re.match(r'^#[A-Fa-f0-9]{3}([A-Fa-f0-9]{3}){0,1}$', color):
```

Comment:
On a second thought, there is no reason for checking `color`, if there is no input, this method is not called. So it'll always evaluate to True. At least that's the behaviour when the field is nullable, it may be different when it's not.